index.js

```diff
@@ -7,7 +7,7 @@ var Reporter = require('jasmine-terminal-reporter');
 var SilentReporter = require('./silent-reporter');
 
 function deleteRequireCache(id) {
-	if (id.indexOf('node_modules') !== -1) {
+	if (!id||id.indexOf('node_modules') !== -1) {
```

Comment:
still not 100%. Just copy paste @kevva's suggstion: https://github.com/sindresorhus/gulp-jasmine/pull/44#discussion_r26506324